capstone/src/components/User/Experience/RegisterExperience/RegisterExperience.jsx

```diff
@@ -75,28 +94,37 @@ export default function RegisterExperience() {
       </Heading>
       {fields.map((inputField) => (
         <FormControl isRequired={inputField.isRequired} key={inputField.id} p="20px">
-          <FormLabel>{inputField.displayText}</FormLabel>
-          {inputField.input === 'input'
-            ? (
-              <Input
-                value={inputField.value}
-                type={inputField.type}
-                placeholder={inputField.placeholder}
-                onChange={(e) => handleRegistryChange(inputField.id, e.target.value)}
-              />
-            )
-            : (
-              <Textarea
-                value={inputField.value}
-                type={inputField.type}
-                placeholder={inputField.placeholder}
-                onChange={(e) => handleRegistryChange(inputField.id, e.target.value)}
-              />
-            )}
+          <FormLabel>
+            {inputField.displayText}
+          </FormLabel>
+          {/* QUESTION: How to refactor that code? */}
+          {inputField.type === 'text' && (
+          <Input
+            value={inputField.value}
+            placeholder={inputField.placeholder}
+            // QUESTION: Mandatory to use e.target.value?
```

Comment:
Most likely required, yeah. When you have a [controlled component](https://reactjs.org/docs/forms.html) it will often expose the current value as `e.target.value`